packages/utils/telemetry-utils/src/config.ts

```diff
@@ -245,20 +254,33 @@ export class CachedConfigProvider implements IConfigProvider {
 }
 
 /**
- * A type containing both a telemetry logger and a configuration provider
+ * A type containing both a telemetry logger and a configuration provider.
+ *
+ * @public
```

Comment:
Hopefully not

Reply:
+1. Confirmed no uses in bohemia nor loopsdk.